src/util/issues.ts

```diff
@@ -1,4 +1,12 @@
 import * as core from '@actions/core';
+import { GitHub } from '@actions/github/lib/utils';
+
+type GitHubClient = InstanceType<typeof GitHub>;
```

Comment:
Should probably dedupe this declaration somewhere.

Reply:
Moved to `src/types.ts`. Should it be in `src/` or `src/util/`?

Reply:
That's fine for now.